### PR TITLE
HTML IFrames 

### LINKED ISSUE #1165: HTML Iframes
<hr>

### HTML IFrames 📜
I would like to contribute in the documentation part of HTML iframes with explanations and examples of how to use it .

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (HTML, CSS, JS)

<hr>

### PR DESCRIPTION
**Pull request by kajaljha09**
<hr>

## Description 📜
Explained about HTML Iframes using code snippets and images.

Fixes #1165

<hr>

## Type of change 📝
Added markdown file of HTML Iframes

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

Annotation 2021-09-30 190058
<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# HTML IFrames (Inline Frames)
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:1]: Space needed after heading. Give a one line space.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# HTML IFrames (Inline Frames) 
+An IFrame (Inline Frame) is an HTML document embedded inside another HTML document on a website.<br>
+The IFrame HTML element is often used to insert content from another source,such as an advertisement, into a Web page.<br>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:3]: Do not use HTML tags, just keep it with Markdown only. Use line spaces around the line to make it as paragraphs.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+<br>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:5]: Remove HTML.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+(Basically it is used to display a web page within a web page)
+
+![iframe image](https://imgur.com/hI0t3ZI.jpg)
+
+
+
+<br>
+<br>
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:14]: Remove empty lines and HTML tags.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+</head>
+<body>
+  
+   <iframe src="demo.html" title="iframe Example 1" width="400" height="300"></iframe>
+    
+</body>
+</html>
+
+```
+<b>Tip</b>: It is a good practice to always include a title attribute for the ```<iframe>```. This is used by screen readers to read out what the content of the iframe is.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:30]: Remove HTML. Use Markdown.

Comment (praveenscience) [reply Web_Development/FrontEnd/HTML_IFrames/iframes.md:30]: Use inline code for iframe not code fence.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:31]: Remove useless empty lines.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+We can use multiple iframes inside a single web page.
+![iframe image](https://imgur.com/9pgJ9LC.jpg)
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:36]: Remove useless empty lines.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+    
+</body>
+</html>
+
+```
+<b>Tip</b>: It is a good practice to always include a title attribute for the ```<iframe>```. This is used by screen readers to read out what the content of the iframe is.
+
+
+We can use multiple iframes inside a single web page.
+![iframe image](https://imgur.com/9pgJ9LC.jpg)
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:34]: Add a line space before.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+
+## Attributes :
+
+* **Height** and **Width** attributes to specify the size of the iframe.
+The height and width are specified in pixels by default.
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:41]: Remove useless empty lines.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+* **Target** attribute of the link must refer to the name attribute of the iframe.
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:43]: Remove useless empty lines.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+* **SRC** attribute refers to the source(url) of the web page to display using iframe.
+<br>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:45]: Remove HTML.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+![img url](https://imgur.com/yuqoXKZ.jpg)
+
+## Removing Default Frameborder :
+The iframe has a border around it by default. However, if you want to modify or remove the iframe borders, the best way is to use the CSS border property.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:50]: Add line space after heading.

Comment (praveenscience) [reply Web_Development/FrontEnd/HTML_IFrames/iframes.md:50]: Remove space before `:`.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+<iframe src="hello.html" style="border:none;"></iframe>
+```
+Similarly, we can use the border property to add the borders of your choice to an iframe. The following example will render the iframe with 2 pixels pink border.
+
+```html
+<iframe src="hello.html" style="border: 2px solid pink;"></iframe>
+```
+
+<br>
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:64]: Remove useless empty lines and HTML tags.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+## Using an iFrame as Link Target :
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:67]: Remove useless empty lines.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+Let's try out an example to understand how it basically works:
+```html
+<iframe src="demo-page.html" name="myFrame"></iframe>
+<p><a href="https://www.google.com" target="myFrame">Open TutorialRepublic.com</a></p>
+```
+
+***Refrences***:
+
+[W3Schools](https://www.w3schools.com)
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:80]: Never quote W3Schools as reference.

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:97]: Remove useless empty lines.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please review the comments.

**Review by praveenscience — APPROVED** [review]
👍

File: Web_Development/FrontEnd/HTML_IFrames/iframes.md [hunk 17]
```diff
@@ -5,8 +5,11 @@ An IFrame (Inline Frame) is an HTML document embedded inside another HTML docume
 The IFrame HTML element is often used to insert content from another source,such as an advertisement, into a Web page.
 
 (Basically it is used to display a web page within a web page)
+
```
Comment (ghulamyazdani) [inline Web_Development/FrontEnd/HTML_IFrames/iframes.md:8]: Can you add you add some more points explaining Iframes?the info is so little

Comment (kajaljha09) [reply Web_Development/FrontEnd/HTML_IFrames/iframes.md:8]: is it fine now @ghulamyazdani ?

**Review by ghulamyazdani — APPROVED** [review]
LGTM 🥳 
Great work @kajaljha09 !!

**Pull request merged**